modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/java/com/liferay/frontend/data/set/sample/web/internal/portlet/FDSSamplePortlet.java

```diff
@@ -129,6 +129,10 @@ private synchronized void _generate(long companyId) throws Exception {
 						ObjectFieldConstants.BUSINESS_TYPE_TEXT,
 						ObjectFieldConstants.DB_TYPE_STRING, true, false, null,
 						"Color", "color", false),
+					ObjectFieldUtil.createObjectField(
```

Comment:
I've added `imageURL` in object definition, so we can map it later to `/image/company_logo`